corehq/warehouse/transforms/sql/user_group_dim.sql

```diff
@@ -13,31 +23,18 @@ SELECT
     gd.id,
     now(),
     now(),
-    combined_user_group_table.deleted
+    false
 FROM
 (
-(
     SELECT
         UNNEST(user_ids) AS user_id,
-        group_id AS group_id,
-        false AS deleted
-    FROM
-        {{ group_staging }}
-)
-
-UNION
-
-(
-    SELECT
-        UNNEST(removed_user_ids) AS removed_user_id,
-        group_id AS group_id,
-        true AS deleted
+        group_id AS group_id
     FROM
         {{ group_staging }}
-)
-) combined_user_group_table
+    WHERE doc_type = 'Group'
```

Comment:
(offlined, going to leave as is for now and address later with intermediate table)